README.md

```diff
@@ -7,12 +7,17 @@ The Azure App Services extension for VS Code lets you quickly browse, create, ma
 * Browse to the Azure Portal for advanced tasks, such as scaling
 * Create new web apps/deployment slots (Linux with Node.js only)
 * Deploy to your web apps/deployment slots
+* View and edit remote files on your web apps (Preview)
 
 ![Deploy to Web App](resources/WebApp_Deploy.png)
 * Start, stop, and restart the web app/deployment slot
 * Swap deployment slots
 * View and edit web app settings
 
+![Create Deployment Slot](resources/Add_Deployment_Slot.png)
```

Comment:
Why are the file extensions 'PNG' uppercase? Haha

This is what it looks like for me:
![screen shot 2017-12-15 at 11 31 34 am](https://user-images.githubusercontent.com/11282622/34057380-932b2c64-e18b-11e7-9db8-6249d8cebb13.png)

Reply:
I have no idea-- that's what my computer defaults to.  One moment please. . .

Reply:
Actually I don't like having these under the 'View and edit web app settings' bullet point anyways. They feel kind of out-of-place. We have to be careful about _too many_ screenshots so that users just ignore all of them. I don't think 'create deployment slot' makes the cut IMO